apps/ledger-live-mobile/src/index.js

```diff
@@ -158,6 +165,11 @@ function App({ importDataString }: AppProps) {
     return null;
   }, []);
 
+  const getPostOnboardingStateChanged = useCallback(
+    (a, b) => a.postOnboarding !== b.postOnboarding,
+    [],
+  );
+
```

Comment:
Hm yeah indeed, I thought this could be good enough as it's coming from a reducer where each action will result in a new object anyway. Having `isEqual` would just filter for cases where an action is dispatched but results in the same data (which for this reducer doesn't really happen).